pkg/operatorsource/purging_test.go

```diff
@@ -51,27 +50,26 @@ func TestReconcile_SpecHasChanged_ReconciliationTriggered(t *testing.T) {
 	assert.Equal(t, nextPhaseWant, nextPhaseGot)
 }
 
-// Use Case: The associated CatalogSourceConfig object is not found while purging.
-// Expected Result: NotFound error is ignored and the next phase should be set
-// to "Validating" so that reconciliation is triggered.
-func TestReconcile_CatalogSourceConfigNotFound_ErrorExpected(t *testing.T) {
+// In the event the associated CatalogSourceConfig object is not found while
+// purging is in progress, we expect NotFound error to be ignored and the next
+// phase set to "Initial" so that reconciliation can start anew.
+func TestReconcileWithPurgingWithCatalogSourceConfigNotFound(t *testing.T) {
```

Comment:
Can we establish a naming convention for tests? Here tests are in pure CamelCase, but in other tests we use a mix of CamelCase and underscores.